src/web/screens/sharingPermissions.tsx

```diff
@@ -91,15 +94,18 @@ function SharingPermissions() {
     }
   };
 
-  const loadSharingList = useCallback(async () => {
-    const response = await GetSharingList();
-    setSharedSiteIds(response.allowed_sites);
-    setSharedTypes(response.allowed_types ?? []);
-  }, []);
-
   useEffect(() => {
+    const loadSharingList = async () => {
+      try {
+        const response = await GetSharingList();
+        setSharedSiteIds(response.allowed_sites);
+        setSharedTypes(response.allowed_types ?? []);
+      } catch (e: unknown) {
+        if (e instanceof ApiError) throwError(e);
+      }
+    };
     loadSharingList();
-  }, [loadSharingList]);
+  }, [throwError]);
```

Comment:
I think we should provide a better error experience here - if there's a 404 from the admin service, we know exactly what's wrong. Just dumping the user to an error where they can't even see the menu any more isn't the right move.